### DISCUSSION TITLE
cannot connect as system/oracle to oracle container

### DISCUSSION THREAD

**Discussion by gusega** [testcontainers/testcontainers-java#7723] [Q&A]
Hi,

I was looking into the [oracle xe container](https://hub.docker.com/r/gvenzl/oracle-xe#:~:text=11.2.0.2%2Dfull%2Dfaststart-,Quick%20Start,-Run%20a%20new).

If I run it from the command line: `docker run -d -p 1521:1521 -e ORACLE_PASSWORD=oracle gvenzl/oracle-xe:21-slim-faststart` I can connect to the database as `system`/`oracle`, but if I use test containers, then I can only connect with a user account `test`/`test`. 

Why is that?

**Comment by eddumelendez** [comment]
If you take a look at the [OracleContainer implementation](https://github.com/testcontainers/testcontainers-java/blob/main/modules/oracle-xe/src/main/java/org/testcontainers/containers/OracleContainer.java#L209-L218) you can notice that some env var are set by default.

When using OracleContainer you can get those values as it is shown below

```java
OracleContainer container = new OracleContainer("gvenzl/oracle-xe:21-slim-faststart");
container.getJdbcUrl();
container.getUsername();
container.getPassword();
```

**Comment by gusega** [comment]
hi @eddumelendez. Thank you, ineed the default password is `test`. So I can connect with `system`/`test`. Thanks a lot. Somehow I completely missed it.
